Subject: Increase minimum stack size for s390 by adding a page.
CC: Patrick Kirsch <pkirsch@suse.de> 

TODO_SIGNOFF

> Wed Jan  9 11:39:16 CET 2008 - yxu@suse.de
> 
> - fixed pthread_create_1_4 (bug 239371)
> - fixed OPEN_MAX missing definition issue
> 
> -------------------------------------------------------------------
> Mon Dec  3 11:43:39 CET 2007 - pkirsch@suse.de
> 
> - temporary fix from npiggin@suse.de for pthread_create_1_6 (#239371)


Quote from Patrick Kirsch:

> So this is like always a temporary fix, because on s390 and s390x the 
> minimal stack size is too minimal. There was no progress or decision to 
> increase the stack size. IIRC, the culprit in this test case is the 
> overhead of the printf call.
> What I did is only the add an extra page.

Author: Patrick Kirsch <pkirsch@suse.de> 

---

--- ltp-full-20070930.orig/testcases/open_posix_testsuite/conformance/interfaces/pthread_create/threads_scenarii.c	2008-01-08 15:18:40.000000000 +0100
+++ ltp-full-20070930/testcases/open_posix_testsuite/conformance/interfaces/pthread_create/threads_scenarii.c	2008-01-08 15:47:31.000000000 +0100
@@ -143,6 +143,7 @@ void scenar_init()
 	output(" min stack size: %li\n", minstacksize);
 	#endif
 	
+	minstacksize = minstacksize + pagesize;	
 	
 	if (minstacksize % pagesize)
 	{
